src/action/info.js

```diff
@@ -26,4 +27,11 @@ class InfoAction {
   }
 }
 
+const calcPercentSynced = response => {
+  const bestHeaderTimestamp = response.best_header_timestamp;
+  const currTimestamp = new Date().getTime() / 1000;
+  const percentSynced = bestHeaderTimestamp * 1.0 / currTimestamp;
+  return percentSynced;
+};
+
```

Comment:
As mentioned, we should probably apply a function that approximates transactions over time, since the loader basically goes to 95% after a few seconds and stays there for minutes. @Roasbeef do you have an idea maybe?